includes/classes/upload.php

```diff
@@ -259,4 +259,13 @@ function message_stack($msg = '', $type = '')
             }
         }
     }
+
+    function sanitizeFileName($filename) {
```

Comment:
- Add type hints and function scope, i.e. `protected function sanitizeFilename(string $filename): string`
- Opening brace on a new line